antarest/study/storage/variantstudy/model/command_listener/command_listener.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024, RTE (https://www.rte-france.com)
+#
+# See AUTHORS.txt
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of the Antares project.
+
+from abc import ABC, abstractmethod
+
+from antarest.core.serialization import AntaresBaseModel
+
+
+class ICommandListener(ABC, AntaresBaseModel, extra="forbid", arbitrary_types_allowed=True):
```

Comment:
why not set the config with the pydantic way like : 
class Config:
        extra = "forbid"
        arbitrary_types_allowed = True
inside the class ?

Reply:
It's the same thing but I believe the way you described was introduced with pydantic v2 so all our code was the way I wrote it until very recently. I can change it but I don't think it really matters